packages/sdk/src/interaction/interaction.js

```diff
@@ -106,9 +107,75 @@ const IX = `{
   }
 }`
 
+const LOGGER_LEVELS = {
+  "debug": 0,
+  "info": 1,
+  "log": 2,
+  "warn": 3,
+  "error": 4
+}
+
+const logger = async (title, message, level) => {
+  const configLoggerLevel =  await config.get("logger.level", LOGGER_LEVELS.debug)
+
+  if (configLoggerLevel >= level) {
+    console.warn(
```

Comment:
Should we `console.error` for level 4, `console.warn` for level 3, and then `console.log` for 0,1,2?

Reply:
There is a `console.debug`